ledger/controllers/surveyor.js

```diff
@@ -415,6 +417,34 @@ v2.phase2 =
     { schema: Joi.object().keys({ submissionId: Joi.string().required().description('verification submissionId') }) }
 }
 
+/*
+   GET /v1/surveyor/{surveyorId}/voterate
```

Comment:
Did you mean `GET /v1/surveyor/voterate/{surveyorType}/{surveyorId}`?  I see that in the routes.

Reply:
yes. can fix